### ISSUE TITLE
This Could Be So Much Better...

### ISSUE THREAD

**Issue by hifihedgehog** [Linaro/works-on-woa#70]
I am genuinely excited for Snapdragon X and I know from the leaker community how it will perform: like a boss. However, I feel this website falls utterly short of the mark for the game-changing hardware it is representing. In its current state, it is not useful. Configuration information and game settings are missing. And this is not just me speaking. This is the same feedback I got from my own forum (https://forum.tabletpc.review) as well as the ROG Ally Discord. From my view, it looks like it was a rushed job to meet a deadline. ProtonDB and ShareDeck are infinitely better and are non-pro community made. Here are some particular pain points that immediately stood out to me:

---

1. The site UX/UI feels unfinished. The Steam Deck community is a good starting point on how to do this the right way and not cut corners or rush things to meet a deadline. Consider seeing these websites for design ideas:
[ShareDeck](https://sharedeck.games/)
[ProtonDB](https://www.protondb.com/)

2. Result submissions form on the website. Making users contribute scores via GitHub pull requests, besides being manual and cumbersome, heavily limits the user participation to solely technical users and specifically those who like using GitHub. If you want to get the most people involved, create a submission form. As an additional protection against users who are trolling or exaggerating with their results, require a screenshot and/or create an applet to verify their hardware.

3. Specifics, specifics, specifics. What resolution? What quality settings? What power plan on the system? Many users are finding this utterly useless and just a PR stunt.

---

In short, this website is the polar opposite of the peak caliber of work that Ryan Shrout and his team at Signal 65 put out for the Surface whitepaper which showcases Snapdragon X Elite's revolutionary performance gains. I have seen these systems perform well with games via leaks that have been fed to me, yet this website is such a rushed and messy effort that it looks embarrassing. In all fairness, this website should be pulled immediately and then republished at later date once it has been refined and organized to a level that will be useful for gamers to gauge performance and that it has a sleek look to it.

**Comment by fahdriyami** [comment]
Agreed. There is a lot of room for improvement to make this a truly community-driven website.

**Comment by Insane-Redditor** [comment]
This website would actually be so much better if they just started updating it. It looks the same as it did day 1, which is not a good look.

**Comment by marcpems** [comment]
Thx for the feedback all. We have been working hard to get the apps data ready and shipped. Now thats done there is more scope for improvements. 
The data is being refreshed and added too as information is available. Please continue to use the https://www.worksonwoa.com/contributing forms (which were added to simplify the process of providing updates for those not able to use github) to share data ... or submit a PR.
